scripts/roles.coffee

```diff
@@ -83,37 +89,43 @@ parseCommaSeparatedString = (string) ->
 removeObjects = (source, itemsToRemove) ->
   _.reject(source, (item) ->
     itemsToRemove.indexOf(item) > -1)
+    
+typeIsArray = Array.isArray || ( value ) -> return {}.toString.call( value ) is '[object Array]'
```

Comment:
won't lie i stole this from stack overflow, i guess I can see if just the left side works by itself